source/optional_features/custom_branding/index.html.md.erb

```diff
@@ -35,10 +41,14 @@ directly](/support_contact_and_more_information/#contact-us).
 
 Colours are customised using hexadecimal representations.
 
-## Reply-to email address
+## Tell your users how to contact you
+
+When you ask GOV.UK Pay to set up custom branding on your [payment confirmation emails](/payment_flow/#confirmation-email), you must tell your users how to contact you.
+
+You can do this using a reply-to email address or another way such as a contact form.
 
-When you ask GOV.UK Pay to set up custom branding on your [payment confirmation emails](/payment_flow/#confirmation-email), you should specify a reply-to email address so your users can contact you.
+If you use a reply-to email address, this email address can be a monitored email inbox for your service, or an unmonitored automated email response.
 
-This reply-to email address can be an email inbox for your service, or an automated response.
+You can customise your monitored and unmonitored email responses in the GOV.UK Pay admin tool using the `customParagraph` field in the payment receipt email template.
```

Comment:
we need to make sure we update the team manual / templates so that this works

Reply:
do we want to say - you should let your user know if the address is unmonitored and provide a way for them to contact your service

Reply:
yes